lib/modelValidator.js

```diff
@@ -279,7 +279,7 @@ function validateValue(key, field, value, models, allowBlankTargets, disallowExt
             }
         }
 
-        var valueErrors = self.valueValidator.validateValue(key, value, field);
+        var valueErrors = self.valueValidator.validateValue(key, value, field, models, allowBlankTargets, disallowExtraProperties);
```

Comment:
It looks like this change is not required as it is into a very different call with the same name on a different object.

The modelValidator.validateValue is using value to mean any object or type

The valueValidator.validateValue is using value to mean any nonObject/noneArray type

Reply:
I'd like to take your changes in but I need them slightly modified to accept them.

Reply:
Ah, sorry, I was out for the weekend and didn't see this until now. Thanks for fixing this in the other branch.